usecases/backup/coordinator.go

```diff
@@ -321,17 +321,28 @@ func (c *coordinator) canCommit(ctx context.Context, method Op, backend string)
 
 // commit tells each participant to commit its backup operation
 // It stores the final result in the provided backend
-func (c *coordinator) commit(ctx context.Context, req *StatusRequest, nodes map[string]string) {
-	c.commitAll(ctx, req, nodes)
+func (c *coordinator) commit(ctx context.Context,
+	req *StatusRequest,
+	nodeMap map[string]string,
+	toleratePartialFailure bool,
+) {
+	var node2Host map[string]string // copy because we need to mutate the map
+	node2Host = make(map[string]string, len(node2Host))
```

Comment:
`len(node2Host)`??
- I'll wager you meant `nodeMap` 😎

Consider adding a hint where / why the map is getting mutated on `#329` (i.e., as fundamental to your change)

Reply:
> I'll wager you meant nodeMap

I'm ok with `node2host`, as it expresses that the mapping is between a given node and its hostname. `nodeMap` seems a bit redundant given the typing system already lets us know this is a `map`.

>Consider adding a hint where / why the map is getting mutated in your comment on #329 (i.e., as fundamental to your change)

Agreed 🙂

Reply:
@noodnik2 you're right I meant `nodeMap (node2Addr)`. Well spotted 👍
Agreed @parkerduckworth 👍. I changed `nodeMap -> node2Addr`